src/modules/esl-trigger/core/esl-trigger.ts

```diff
@@ -101,6 +70,28 @@ export class ESLTrigger extends ESLBaseElement {
     return this.a11yTarget ? this.querySelector(this.a11yTarget) : this;
   }
 
+  /** Marker to allow track hover */
+  public get allowHover() {
+    return DeviceDetector.hasHover && ESLMediaQuery.for(this.trackHover).matches;
+  }
+  /** Marker to allow track clicks */
+  public get allowClick() {
+    return ESLMediaQuery.for(this.trackClick).matches;
+  }
+
+  private static parseDelayValue(delay: string): number | undefined {
+    return isNaN(+delay) ? undefined : +delay;
+  }
+  /** Show delay attribute processing */
+  public get showDelayValue(): number | undefined {
+    return ESLMediaRuleList.parse(this.showDelay, ESLTrigger.parseDelayValue).activeValue;
+  }
+  /** Hide delay attribute processing */
+  public get hideDelayValue(): number | undefined {
+    const hideDelay = +(this.hideDelay || 'none');
+    return isNaN(hideDelay) ? undefined : hideDelay;
+  }
```

Comment:
Maybe we can use ESLTrigger.parseDelayValue(this.hideDelay || 'none') here?

Reply:
Yea, good catch. Thanks!